src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -36,6 +36,13 @@
 var DynamicTableHeaderContextMenuClass = null;
 var ProgressColumnWidth = -1;
 
+const tofixedPointString = function(number, digits)
```

Comment:
Can we move this function to `misc.js`?

Reply:
I don't think so, this file doesn't have access to misc.js AFAIK.

Reply:
`dynamicTable.js` calls some of the other functions in `misc.js`, so I think it may work.

Reply:
moved.